manager-bundle/src/Resources/skeleton/public/.htaccess

```diff
@@ -39,6 +39,7 @@
     #   following RewriteCond (best solution)
     RewriteCond %{ENV:REDIRECT_STATUS} ^$
     RewriteRule ^index\.php(?:/(.*)|$) %{ENV:BASE}/$1 [R=301,L]
+    RewriteRule ^app\.php(?:/(.*)|$) %{ENV:BASE}/$1 [R=301,L]
```

Comment:
Isn’t there a `RewriteCond` missing?

```suggestion
    RewriteCond %{ENV:REDIRECT_STATUS} ^$
    RewriteRule ^app\.php(?:/(.*)|$) %{ENV:BASE}/$1 [R=301,L]
```

Reply:
Can we change it to ` RewriteRule ^(index|app)\.php(?:/(.*)|$) %{ENV:BASE}/$1 [R=301,L]`?

Reply:
@leofeyer yes, but it has to be `^(?:index|app)\.php(?:/(.*)|$)`, otherwise the `$1` would be incorrect.

Reply:
See #6307